### PR TITLE
Fixed Installation instructions

### PR DESCRIPTION
**Pull request by omichel**
Use the https protocol instead of http.
The signature file is mandatory, not optional.
Improved English.

👍

### CODE REVIEW

File: docs/guide/installation-procedure.md [hunk 1]
```diff
@@ -41,25 +41,26 @@ First of all, you may want to configure your APT package manager by adding the C
...
-When you will close the window, the APT packages list should be automatically updated.
+When you close the window, the APT packages list should be automatically updated.
 Otherwise you can manually execute the following command:
 
 ```sh
 apt-get update
 ```
 
-Optionally, Webots can be authentified thanks to the `Cyberbotics.asc` signature file which can be downloaded [here](http://www.cyberbotics.com/linux), using this command:
+Webots should be authenticated with the [Cyberbotics.asc](https://www.cyberbotics.com/Cyberbotics.asc) signature file which can be downloaded from the [Webots download page](https://www.cyberbotics.com/download), and installed using this command:
```
Comment (DavidMansolino) [inline docs/guide/installation-procedure.md]: This should probably go before the `apt-get update`, otherwise, this commands shows warnings.

Comment (DavidMansolino) [inline docs/guide/installation-procedure.md]: > which can be downloaded from the [Webots download page](https://www.cyberbotics.com/download),

Is this still useful now that previous sentence is a link?

Comment (omichel) [reply docs/guide/installation-procedure.md]: Yes, I believe it's nice to also see the page where it's embedded.

Comment (omichel) [reply docs/guide/installation-procedure.md]: Yes, I believe it's nice to see the page where it's embedded.

**Comment by omichel** [comment]
👍

**Pull request merged**